<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../melampig.qrc">
    <normaloff>:/icons/user_gray.png</normaloff>:/icons/user_gray.png</iconset>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>663</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="actionClose_All"/>
    <addaction name="separator"/>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
    <addaction name="separator"/>
    <addaction name="actionArrange_icons"/>
    <addaction name="separator"/>
    <addaction name="action_Next"/>
    <addaction name="action_Previos"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="toolTip">
    <string>File</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
   <iconset resource="../../../melampig.qrc">
     <normaloff>:/end</normaloff>:/end</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="iconText">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
   <iconset resource="../../../melampig.qrc">
     <normaloff>:/icons/cancel.png</normaloff>:/icons/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Cl&amp;ose</string>
   </property>
   <property name="shortcut">
    <string>Alt+W</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>Close &amp;All</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="icon">
   <iconset resource="../../../melampig.qrc">
     <normaloff>:/icons/application_tile_horizontal.png</normaloff>:/icons/application_tile_horizontal.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Tile</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="icon">
   <iconset resource="../../../melampig.qrc">
     <normaloff>:/icons/application_cascade.png</normaloff>:/icons/application_cascade.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cascade</string>
   </property>
  </action>
  <action name="actionArrange_icons">
   <property name="text">
    <string>Arrange &amp;icons</string>
   </property>
  </action>
  <action name="action_Next">
   <property name="icon">
   <iconset resource="../../../melampig.qrc">
     <normaloff>:/icons/arrow_right.png</normaloff>:/icons/arrow_right.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Next</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="action_Previos">
   <property name="icon">
   <iconset resource="../../../melampig.qrc">
     <normaloff>:/icons/arrow_left.png</normaloff>:/icons/arrow_left.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Previos</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
   <iconset resource="../../../melampig.qrc">
     <normaloff>:/icons/information.png</normaloff>:/icons/information.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
   <iconset resource="../../../melampig.qrc">
     <normaloff>:/icons/qt-logo.png</normaloff>:/icons/qt-logo.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../melampig.qrc"/>
 </resources>
 <connections/>
</ui>
